pkg/xds/topology/router.go

```diff
@@ -60,32 +60,55 @@ func BuildRouteMap(dataplane *mesh_core.DataplaneResource, routes []*mesh_core.T
 	for _, oface := range dataplane.Spec.Networking.GetOutbound() {
 		serviceName := oface.GetTagsIncludingLegacy()[mesh_proto.ServiceTag]
 		policy, exists := policyMap[serviceName]
+
 		outbound := dataplane.Spec.Networking.ToOutboundInterface(oface)
 		if exists {
-			routeMap[outbound] = policy.(*mesh_core.TrafficRouteResource)
-		} else {
-			routeMap[outbound] = &mesh_core.TrafficRouteResource{
-				Meta: &pseudoMeta{
-					Name: "(implicit default route)",
-				},
-				Spec: mesh_proto.TrafficRoute{
-					Sources: []*mesh_proto.Selector{{
-						Match: mesh_proto.MatchAnyService(),
-					}},
-					Destinations: []*mesh_proto.Selector{{
-						Match: mesh_proto.MatchService(serviceName),
-					}},
-					Conf: []*mesh_proto.TrafficRoute_WeightedDestination{{
-						Weight:      100,
-						Destination: mesh_proto.MatchTags(oface.GetTagsIncludingLegacy()),
-					}},
-				},
+			route := policy.(*mesh_core.TrafficRouteResource)
+			if route.Spec.GetConf().HasWildcard() {
```

Comment:
I don't think this should rely on wildcard here. How about this case

Traffic Route
```
conf:
  split:
  - weight: 100
    tags:
      kuma.io/service: 'backend'
```
+ Outbound
```
outbound:
- port: 1234
  tags:
    kuma.io/service: backend
    version: 2
```
=
```
conf:
  split:
  - weight: 100
    tags:
      kuma.io/service: 'backend'
      version: 2
```